client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/hotspot/ChangeHotspotStatusParams.java

```diff
@@ -19,14 +19,12 @@
  */
 package org.sonarsource.sonarlint.core.clientapi.backend.hotspot;
 
-import org.eclipse.lsp4j.jsonrpc.validation.NonNull;
-
 public class ChangeHotspotStatusParams {
   private final String configurationScopeId;
   private final String hotspotKey;
   private final HotspotStatus newStatus;
 
-  public ChangeHotspotStatusParams(@NonNull String configurationScopeId, @NonNull String hotspotKey, @NonNull HotspotStatus newStatus) {
```

Comment:
As discussed, these `@NonNull` annotations are specific to lsp4j and don't have the same meaning as what is set in the `package-info.java`. As long as we are still running SLCORE in-process it's not very important but it will be when moving out of process

Reply:
Addressed in [e3eedf4](https://github.com/SonarSource/sonarlint-core/pull/612/commits/e3eedf4a8d30bebab82f68b85844e20e2314b3f0)